### PR TITLE
Fix compass, hidden mode for GUI (key "U") and menu screen with loading

### PR DESCRIPTION
**Pull request by adam-kucz**

### CODE REVIEW

File: Scripts/Player.cs [hunk 1]
```diff
@@ -89,6 +89,31 @@ private set
...
+        noGUIMode = false;
+
+        Inventories = new Dictionary<Item.ItemType, Inventory>
+        {
+            [Item.ItemType.CONSUMABLE] = new Inventory(Item.ItemType.CONSUMABLE, PLAYER_INVENTORY_COUNT),
+            [Item.ItemType.PROCESSED] = new Inventory(Item.ItemType.PROCESSED, PLAYER_INVENTORY_COUNT),
+            [Item.ItemType.BLOCK] = new Inventory(Item.ItemType.BLOCK, PLAYER_INVENTORY_COUNT)
+        };
+
+        AddItem(ItemStorage.items[ItemID.CAKE], 3);
```
Comment (JamesHinshelwood) [inline Scripts/Player.cs]: These are already included at line 169. Please remove before merging.

**Review by JamesHinshelwood — APPROVED** [review]
:accept:

**Comment by adam-kucz** [comment]
Requesting comments on new commit

File: Scripts/Inventory/ItemStorage.cs [hunk 2]
```diff
@@ -4,47 +4,41 @@
...
-    private static readonly IDictionary<string, Texture> texDict;
+    public Item this[ItemID id]
+    {
+        get
+        {
+            return itemResourceLoader != null ? items[id] : throw new Exception("items are not initialized yet");
+        }
+    }
+
+    public static ItemStorage Instance { get; private set; }
```
Comment (JamesHinshelwood) [inline Scripts/Inventory/ItemStorage.cs:23]: eww singletons?

**Review by JamesHinshelwood — APPROVED** [review]
Nice

File: Scripts/Inventory/ItemStorage.cs (hunk 2 above)
Comment (adam-kucz) [reply Scripts/Inventory/ItemStorage.cs:23]: I know 😭 , see comment earlier

**Pull request merged**